packages/types/src/signUpFuture.ts

```diff
@@ -2,21 +2,32 @@ import type { SetActiveNavigate } from './clerk';
 import type { PhoneCodeChannel } from './phoneCodeChannel';
 import type { SignUpIdentificationField, SignUpStatus } from './signUpCommon';
 
-export interface SignUpFutureCreateParams {
+interface SignUpFutureAdditionalParams {
+  firstName?: string;
+  lastName?: string;
+  unsafeMetadata?: SignUpUnsafeMetadata;
+  legalAccepted?: boolean;
+}
+
+export interface SignUpFutureCreateParams extends SignUpFutureAdditionalParams {
```

Comment:
Good thinking keeping this internal for now. I think we might want to consolidate the future + current params at some point so we don't have duplicate definitions. Not sure how this will look like yet, so lets keep what we can private ✔️